RFCs/0018-Skip-Default-Extensions.md

```diff
@@ -0,0 +1,34 @@
+# 0018 Skip Default Extensions
+
+## Summary
+This note outlines the proposed changes for adding option for skipping lookup and initialization of default extensions.
+
+## Motivation
+In addition to test adapters provided via TestAdaptersPaths, test platform requires default adapters present in Extensions folder. This folder lies where vstest.console.exe lies. Even if default extensions can't run/discover tests, we still pass test assembly to default extensions and thus increase adapter lookup and initialization cost. 
```

Comment:
In addition to test adapters provided via TestAdaptersPaths, test platform loads default adapters present in Extensions folder. This folder is present alongside vstest.console.exe. Even for test sources where default extensions can't run/discover tests, we still pass test source assembly to them, this causes both increase adapter lookup, their initialization cost, & adds overhead to execution time.

Reply:
what do we mean by adapter lookup?
Do we mean adapter load ? or finding the Executor type?

Reply:
Adapter lookup means finding the executor type.

Reply:
Renamed the adapter lookup term